llvm/test/Transforms/DeadArgElim/sycl-kernels-neg2.ll

```diff
@@ -0,0 +1,14 @@
+; RUN: rm -rf %t && mkdir -p %t
```

Comment:
What is this command for? What is the reason for using temporal directory instead of temporal file?

"%t/bad_file.h" -> "%t-bad_file.h"

Reply:
Fixed.